onnxruntime/test/shared_lib/test_inference.cc

```diff
@@ -346,11 +346,15 @@ TEST(CApiTest, DISABLED_test_custom_op_library) {
 
   std::string lib_name;
 #if defined(_WIN32)
-  lib_name = "custom_op_library.dll";
+  char current_directory[/*MAX_PATH*/ 260];
+  if (_getcwd(current_directory, _countof(current_directory)))
+    lib_name = current_directory;
+  lib_name += "\\custom_op_library.dll";
+
 #elif defined(__APPLE__)
   lib_name = "libcustom_op_library.dylib";
 #else
-  lib_name = "./libcustom_op_library.so";
+lib_name = "./libcustom_op_library.so";
```

Comment:
If we agree that absolute paths is the way to go, we should be consistent for all platforms and document this? Can be a separate PR perhaps.

Reply:
Yeah, I have another bug about unloading custom op libraries so I'll address it there.